### PR TITLE
Update student profiles

### PR DESCRIPTION
**Pull request by skmorgane**
Moves graduated students to alumni, adds websites, fixes typos, updates profiles.

### CODE REVIEW

**Comment by skmorgane** [comment]
I checked out the error and it may have something to do with different versions of Hugo being used on netlify vs. locally? https://github.com/gcushen/hugo-academic/issues/1227

File: content/people/alumni/kristina-riemer.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ title = "Kristina Riemer"
 summary = "PhD Student"
 
 # Optional image to display on homepage (relative to `static/img/` folder).
-image_preview = "kristina-riemer.jpg"
+image_preview = "alumni/kristina-riemer.jpg"
 
 # Tags: can be used for filtering projects.
 # Example: `tags = ["ernest-lab", "white-lab"]`
-tags = ["white-lab"]
+tags = [alumni"]
```
Comment (henrykironde) [inline content/people/alumni/kristina-riemer.md]: looks like we missed "

Comment (ethanwhite) [reply content/people/alumni/kristina-riemer.md]: Thanks for tracking that down @henrykironde!

**Comment by skmorgane** [comment]
Bah. Quotation marks! Thanks for catching that, @henrykironde !

**Pull request merged**